tensorboard/backend/http_util.py

```diff
@@ -184,6 +189,8 @@ def Respond(
     headers.append(("X-Content-Type-Options", "nosniff"))
     if content_encoding:
         headers.append(("Content-Encoding", content_encoding))
+    if filename:
+        headers.append(("Content-Disposition", 'filename="%s"' % filename))
```

Comment:
Want `attachment; filename=…`?

Reply:
So I specifically left out `attachment` since with the current behavior it's possible in Chrome to open the .json file in the browser in a new tab if you want (the `.csv` is always forcibly downloaded; I think this is content-type specific browser behavior).  As a user I like it when it's possible to do this rather than requiring that I always download the file, so that's why I did it this way.

Setting the `download` attribute in the HTML tag but omitting `attachment` means that the behavior depends on whether you just click the link (download) or click to open in a new tab (browser dependent, but may stay in the browser), so that seemed like a nice option to preserve.

That said I agree it's overall a bit less consistent to reason about, so if you prefer I'm happy to add `attachment`.

Reply:
That’s fine with me (I also like that behavior). In that case, let’s fix
the newly added docs to not mention `Content-Disposition: attachment`?